frontend/src/reducers/auth.js

```diff
@@ -0,0 +1,30 @@
+import { LOGIN_SUCCESS } from '../actions/types';
+
+const initialState = {
+  access: localStorage.getItem('access'),
+  refresh: localStorage.getItem('refresh'),
+  isLoading: false,
+
+  // For now Munally set. Once functionality done with authentication set "isAuthenticated: false"
+  isAuthenticated: true,
+  user: null,
+};
+
+export default function foo(state = initialState, action) {
```

Comment:
Maybe rename from foo

Reply:
What name should put? I guess there were no name of function in default but now new version of react gives warning.
```
Compiled with warnings.
Unexpected default export of anonymous function  import/no-anonymous-default-export

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

Reply:
authReducer